ui/react-ui/src/components/routes/reports/functionReport/FunctionUserIdReport.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from 'react';
+import { JsonToTable } from "react-json-to-table";
+
+function FunctionUserIdReport() {
+    const [jsonReport, setJsonReport] = useState();
+    const [showJson, setshowJson] = useState(false);
+    const jsonUrl = "https://nsc-fun-dev-usw2-thursday.azurewebsites.net/api/users";
```

Comment:
seems you are implementing /users instead of /users/{user_id}

Reply:
Solved it now...thanks:-)